### ISSUE TITLE
[Tensorflow] Wrapping tensorflow/core/graph/graph_constructor.h

### ISSUE THREAD

**Issue by pcmoritz** [bytedeco/javacpp-presets#165]
Thanks a lot for your help so far!

To make modifying existing TensorFlow graphs in Java possible we'd like to wrap
```C++
extern Status ConvertGraphDefToGraph(const GraphConstructorOptions& opts,
                                     const GraphDef& gdef, Graph* g);
```
Unfortunately for this to work I think we also need
```C++
std::function<bool(const Node*)> cse_consider_function = nullptr;
```
from the structs in the same file. I tried
```Java
.put(new Info("std::function<bool(const Node*)>").pointerTypes("NodePtrFunc").define())
```
and then
```Java
    public static class NodePtrFunc extends FunctionPointer {
        static { Loader.load(); }
        /** Pointer cast constructor. Invokes {@link Pointer#Pointer(Pointer)}.\                                                                          
         */
        public    NodePtrFunc(Pointer p) { super(p); }
        protected NodePtrFunc() { allocate(); }
        private native void allocate();
        public native @Cast("bool") boolean call(Pointer ptr);
    }
```
Unfortunately this doesn't seem to work, the Parser seems to have problems with the `const` (it seems to contract `const Node*` to `constNode*`). Any idea how to fix this?

Best,
Philipp.

**Comment by saudet** [comment]
I've fixed the issue with the `const`, but that's not enough. I've made those members "write only". We can't convert `std::function` to a function object or a function pointer like JavaCPP uses right now... Anyway, the consider function looks like a setting for users, right? So, should be OK the way I've put it.

**Comment by saudet** [comment]
This has been included in the release of version 1.2, so I'm guessing that the above restrictions are satisfactory. Let me know if they are not though, thanks!
